installer/datafiles/ruby.data

```diff
@@ -7699,7 +7698,7 @@ ${{RUBY_DEST}}/lib/ruby/gems/2.6.0/gems/fluentd-0.12.40/lib/fluent/plugin/in_mon
 ${{RUBY_DEST}}/lib/ruby/gems/2.6.0/gems/fluentd-0.12.40/lib/fluent/plugin/in_dummy.rb; ${{RUBY_INT}}/lib/ruby/gems/2.6.0/gems/fluentd-0.12.40/lib/fluent/plugin/in_dummy.rb; 664; root; root
 ${{RUBY_DEST}}/lib/ruby/gems/2.6.0/gems/fluentd-0.12.40/lib/fluent/plugin/filter_stdout.rb; ${{RUBY_INT}}/lib/ruby/gems/2.6.0/gems/fluentd-0.12.40/lib/fluent/plugin/filter_stdout.rb; 664; root; root
 ${{RUBY_DEST}}/lib/ruby/gems/2.6.0/gems/fluentd-0.12.40/lib/fluent/plugin/in_exec.rb; ${{RUBY_INT}}/lib/ruby/gems/2.6.0/gems/fluentd-0.12.40/lib/fluent/plugin/in_exec.rb; 644; root; root
-${{RUBY_DEST}}/lib/ruby/gems/2.6.0/gems/fluentd-0.12.40/lib/fluent/plugin/filter_parser.rb; ${{RUBY_INT}}/lib/ruby/gems/2.6.0/gems/fluentd-0.12.40/lib/fluent/plugin/filter_parser.rb; 664; root; root
+${{RUBY_DEST}}/lib/ruby/gems/2.6.0/gems/fluentd-0.12.40/lib/fluent/plugin/filter_parser.rb; ${{RUBY_INT}}/lib/ruby/gems/2.6.0/gems/fluentd-0.12.40/lib/fluent/plugin/filter_parser.rb; 644; root; root
```

Comment:
Why is this changed? Ruby shouldn't have any impact on fluentd plugin perms

Reply:
Fluentd is part of ruby gems

Reply:
I think the patch Ashley created caused the permission to change from 664 to 644

Reply:
but that is fine, we have the same permission with in_exec.rb and in_syslog.rb where we added a patch for that.

```
${{RUBY_DEST}}/lib/ruby/gems/2.6.0/gems/fluentd-0.12.40/lib/fluent/plugin/in_syslog.rb; ${{RUBY_INT}}/lib/ruby/gems/2.6.0/gems/fluentd-0.12.40/lib/fluent/plugin/in_syslog.rb; 644; root; root
${{RUBY_DEST}}/lib/ruby/gems/2.6.0/gems/fluentd-0.12.40/lib/fluent/plugin/in_exec.rb; ${{RUBY_INT}}/lib/ruby/gems/2.6.0/gems/fluentd-0.12.40/lib/fluent/plugin/in_exec.rb; 644; root; root
${{RUBY_DEST}}/lib/ruby/gems/2.6.0/gems/fluentd-0.12.40/lib/fluent/plugin/filter_parser.rb; ${{RUBY_INT}}/lib/ruby/gems/2.6.0/gems/fluentd-0.12.40/lib/fluent/plugin/filter_parser.rb; 644; root; root
```